### PR TITLE
Add invoices docs

### PR DESCRIPTION
**Pull request by lkostrowski**
This adds missing functional docs for Invoices app, based on the [template](https://github.com/saleor/saleor-docs/pull/829)

### CODE REVIEW

File: docs/developer/app-store/apps/invoices.mdx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+title: Invoices
+sidebar_position: 4
+---
+
+# Invoices
+
+## Metadata
+
+- Saleor version: 3.10
+- Roadmap https://github.com/saleor/apps/labels/App%3A%20Invoices
```
Comment (peelar) [inline docs/developer/app-store/apps/invoices.mdx:11]:
```suggestion
- [Roadmap](https://github.com/saleor/apps/labels/App%3A%20Invoices)
```

File: docs/developer/app-store/apps/invoices.mdx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+- Link to dashboard
+- Link to GitHub
```
Comment (peelar) [inline docs/developer/app-store/apps/invoices.mdx:13]: **suggestion (blocking)**: provide links

File: docs/developer/app-store/apps/invoices.mdx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    App->>+Saleor: Upload a PDF file
+    Saleor->>+App: Provide a link to the PDF file
+    App->>+Saleor: Attach a link to the PDF file to the Order
+```
+
+## Configuration
+
+The app allows to configure a shop address per each channel separately.
+
+If the channel is not configured, the app will fetch the Shop address from settings and use it.
```
Comment (peelar) [inline docs/developer/app-store/apps/invoices.mdx:59]: **suggestion (non-blocking)**: Maybe we can point to where exactly the settings come from (what query field or where in dashboard).

File: docs/developer/app-store/apps/invoices.mdx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+1. Ensure the "Invoices" plugin is disabled; otherwise, duplicated invoices can be generated.
+2. Ensure the App is installed and enabled.
+3. Optionally configure the custom address. If the custom address is not configured, fill shop address in the Dashboard settings.
+4. Find any order in the Dashboard, find the "Invoices" section, and click the "Generate" button.
+5. Wait a couple of seconds and refresh the dashboard.
+6. The invoice should be attached to the order in the "Invoices" section.
+
+## Troubleshooting
+
+- During local development with Saleor running locally, there are file uploading issues; see [this issue](https://github.com/saleor/apps/issues/285).
```
Comment (peelar) [inline docs/developer/app-store/apps/invoices.mdx:74]:
```suggestion
- During development with Saleor running locally, there are file uploading issues; see [this issue](https://github.com/saleor/apps/issues/285).
```

File: docs/developer/app-store/apps/invoices.mdx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+Check [Github issues](https://github.com/saleor/apps/labels/App%3A%20Invoices) to see up to date list of known issues or to report a new one.
+
+## Development
+
+Visit the development guide to learn more about App Store app development.
```
Comment (peelar) [inline docs/developer/app-store/apps/invoices.mdx:80]: **suggestion (blocking)**: add missing link

**Review by peelar — CHANGES_REQUESTED** [review]

File: docs/developer/app-store/apps/invoices.mdx (hunk 1 above)
Comment (lkostrowski) [reply docs/developer/app-store/apps/invoices.mdx:11]: Thats WIP, I will try to adda react component

File: docs/developer/app-store/apps/invoices.mdx (hunk 2 above)
Comment (lkostrowski) [reply docs/developer/app-store/apps/invoices.mdx:13]: yea

File: docs/developer/app-store/apps/invoices.mdx (hunk 5 above)
Comment (lkostrowski) [reply docs/developer/app-store/apps/invoices.mdx:80]: I think we dont have it :D

File: docs/developer/app-store/apps/invoices.mdx (hunk 1 above)
Comment (lkostrowski) [reply docs/developer/app-store/apps/invoices.mdx]: I added but I'm going to ask for a design

File: docs/developer/app-store/apps/invoices.mdx (hunk 5 above)
Comment (lkostrowski) [reply docs/developer/app-store/apps/invoices.mdx:81]: Why this page is called "app examples"?

Comment (lkostrowski) [reply docs/developer/app-store/apps/invoices.mdx:81]: added

File: docs/developer/app-store/apps/invoices.mdx (hunk 3 above)
Comment (lkostrowski) [reply docs/developer/app-store/apps/invoices.mdx:60]: done

File: docs/developer/app-store/apps/invoices.mdx (hunk 4 above)
Comment (lkostrowski) [reply docs/developer/app-store/apps/invoices.mdx]: done

File: components/AppMetadata/AppMetadata.tsx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+        </span>
+      </div>
+      <div>
+        <span>
+          Roadmap: <a href={roadmapUrl}>GitHub</a>
+        </span>
+      </div>
+      <div>
+        <span>
+          Repository: <a href={githubUrl}>GitHub</a>
```
Comment (peelar) [inline components/AppMetadata/AppMetadata.tsx:29]: **suggestion (non-blocking)**: add `target="_blank"`

Comment (peelar) [reply components/AppMetadata/AppMetadata.tsx:29]: **suggestion (non-blocking)**: IMO, it's more aesthetic. 

```suggestion
          <a href={githubUrl}>Repository</a>
```

**Review by peelar — APPROVED** [review]

File: components/AppMetadata/AppMetadata.tsx (hunk 6 above)
Comment (lkostrowski) [reply components/AppMetadata/AppMetadata.tsx:29]: I'd wait for a design of the component

File: docs/developer/app-store/apps/invoices.mdx (hunk 3 above)
Comment (2can) [reply docs/developer/app-store/apps/invoices.mdx:60]: please make the `Shop` a link, the same as in Features

File: docs/developer/app-store/apps/invoices.mdx [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+
+## Introduction
+
+The Invoices app allows you to generate simple PDF invoice files for every order in your store.
+
+## Features
+
+Generates a PDF file when an invoice is requested (in Dashboard or with an API). Then attaches it to the order.
+
+The invoice shop address can be inherited from the [Shop](https://docs.saleor.io/docs/3.x/api-storefront/miscellaneous/objects/shop) settings or configured manually per channel.
```
Comment (2can) [inline docs/developer/app-store/apps/invoices.mdx:24]: Docusaurus won't validate that link, please make it relative.

**Review by 2can — CHANGES_REQUESTED** [review]

File: docs/developer/app-store/apps/invoices.mdx (hunk 7 above)
Comment (lkostrowski) [reply docs/developer/app-store/apps/invoices.mdx]: done

**Review by 2can — APPROVED** [review]

**Pull request merged**